.github/workflows/ci.yml

```diff
@@ -53,9 +62,19 @@ jobs:
           key: ${{ runner.os }}-gradle-home-${{matrix.gradle_args}}_check-${{ hashFiles('**/*.gradle') }}
       - name: Clear existing docker image cache
         run: docker image prune -af
+      - name: Thundra Gradle Test Instrumentation
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        uses: thundra-io/thundra-gradle-test-action@v1
+        with:
+          apikey: ${{ secrets.THUNDRA_API_KEY }}
+          project_id: ${{ secrets.THUNDRA_PROJECT_ID }}
       - name: Build and test with Gradle (${{matrix.gradle_args}})
         run: |
-          ./gradlew --no-daemon --continue --scan ${{matrix.gradle_args}}
+          ./gradlew --no-daemon --continue --scan ${{matrix.gradle_args}} \
+            $($THUNDRA_GRADLE_INIT_SCRIPT_PATH || echo "--init-script $THUNDRA_GRADLE_INIT_SCRIPT_PATH")
+        env:
+          THUNDRA_AGENT_TEST_RUN_ID: ${{ needs.set_thundra_testrun_id.outputs.thundra_agent_testrun_id }}
+          THUNDRA_AGENT_REPORT_REST_BASEURL: https://collector.thundra.us/v1
```

Comment:
This base URL is our staging environment. Our showroom page is fetching the data from this environment. Normally, in production, you don't need to set this. We wanted to split the showroom environment from production, that's why this is needed here.